ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -126,7 +127,11 @@ object LedgerApiTestTool {
     }
 
     val performanceTestsToRun =
-      Tests.performanceTests(config.performanceTestsReport).filterKeys(config.performanceTests)
+      Tests
+        .performanceTests(config.performanceTestsReport)
+        .view
+        .filterKeys(config.performanceTests)
+        .toMap
```

Comment:
```suggestion
      Tests.performanceTests(config.performanceTestsReport).filterKeys(config.performanceTests)
```
Am I overlooking something or can we just skip the laziness here, since there's only one transformation?

Reply:
`filterKeys` is deprecated https://www.scala-lang.org/api/current/scala/collection/immutable/Map.html#filterKeys(p:K=%3EBoolean):scala.collection.MapView[K,V]

Reply:
Uh uh, thanks!